jetty-server/src/test/java/org/eclipse/jetty/server/ForwardedRequestCustomizerTest.java

```diff
@@ -199,7 +199,7 @@ public static Stream<Arguments> cases()
             Arguments.of(new Request("IPv4 in Request Line")
                     .headers(
                         "GET https://1.2.3.4:2222/ HTTP/1.1",
-                        "Host: wrong"
+                        "Host: 1.2.3.4:2222"
```

Comment:
Uhm. The point of this test was to make sure the authority was extracted from the URI, not the `Host` header.
Perhaps it should be moved to a different test that allows `MISMATCHED_AUTHORITY`.